src/server/frontend_xwayland/xwayland_wm_shellsurface.cpp

```diff
@@ -37,12 +37,11 @@ mf::XWaylandWMShellSurface::XWaylandWMShellSurface(wl_client* client,
                                                    OutputManager* const output_manager)
     : WindowWlSurfaceRole{&seat, client, surface, shell, output_manager}
 {
-    params->type = mir_window_type_normal;
+//    params->type = mir_window_type_normal;
```

Comment:
This is still a bit unclear if we should be a normal type or not, since we dont have windows borders in x11.

Reply:
not 100% how this works in mir, but we *can* make x11 borders, but i think mir would be an better option for this since x11 can tell us when to draw borders or not

Reply:
The code doesn't compile and isn't essential at this stage. Left as a comment to remind us to fix later.